doc/api.rst

```diff
@@ -411,3 +411,15 @@ Plotting
    plot.imshow
    plot.line
    plot.pcolormesh
+   plot.FacetGrid
+
+FacetGrid methods
```

Comment:
one more suggestion: https://github.com/clarkfitzg/xray/pull/2

we can remove this methods and let them just show up on the doc page for FacetGrid.